.gitignore

```diff
@@ -8,3 +8,4 @@ __pycache__/
 *.py[cod]
 .idea
 .vscode/
+.mypy_cache
```

Comment:
Can we use our template for `.gitignore` here? It will require an update because mypy is not there.

Reply:
Updated and applied.